<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_dataset_widget</class>
 <widget class="QWidget" name="create_dataset_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Содание Коллекции Данных</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="bottom_message">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="name_edit"/>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="learn_data_button">
       <property name="text">
        <string>Данные для обучения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="test_data_button">
       <property name="text">
        <string>Данные для валидации</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="top_message">
     <property name="text">
      <string>Выберите два набора данных:
 - Для обучения классификатора 
 - Для валидации результатов</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="create_dataset_button">
     <property name="text">
      <string>Создать</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="name_label">
     <property name="text">
      <string>Введите имя новой коллекции:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
